website/docs/reference/datasources/hubspot.md

```diff
@@ -0,0 +1,151 @@
+---
+sidebar_position: 10
+---
+
+# HubSpot
+
+
+HubSpot is a **cloud-based CRM** designed to help align sales and marketing teams, promote sales, increase ROI, and improve your inbound marketing strategy to produce more qualified leads. In simple words, it's a software platform created to assist your business with marketing and sales.
+
+:::info
+To integrate HubSpot with Appsmith, you should understand the [basics of connecting to datasource](/core-concepts/connecting-to-data-sources/) on Appsmith.
```

Comment:
@harshilp24 Also had requested to remove **basics of** from the hyperlinked text. Only hyperlink connecting to datasource. Thanks :)

Reply:
done